adyenv6b2ccheckoutaddon/src/com/adyen/v6/facades/DefaultAdyenCheckoutFacade.java

```diff
@@ -350,7 +353,7 @@ public OrderData handle3DResponse(final HttpServletRequest request) throws Excep
 
             throw new AdyenNonAuthorizedPaymentException(paymentResult);
         } catch (ApiException e) {
-            LOGGER.error("API Exception " + e.getError());
+            LOGGER.error("API Exception", e);
             throw e;
```

Comment:
Usually if you catch an exception it's better to handle it right away 
Right now we log the exception then throw it again so next time if another function catches it it can happen that for the same error we will have 2 logs - I would let the exception bubble up and catch it where I can actually handle it and log it also then or handle it here and not throwing the exception after logging